openid-sharedsignals-framework-1_0.md

```diff
@@ -671,17 +681,14 @@ Content-Type: application/json
   "verification_endpoint":
     "https://tr.example.com/ssf/mgmt/verification",
   "critical_subject_members": [ "tenant", "user" ],
-  "supported_scopes":
-      {
-        "status_endpoint": ["status_scope"],
-        "configuration_endpoint": ["admin_scope", "status_scope"]
-      },
-  "authorization_servers": [
+  "authorization_schemes": {
+    [
       {
-          "scopes": ["admin_scope", "status_scope"],
-          "servers": ["https://myauthzserver.example"]
+        "type": "oauth2",
```

Comment:
where are these types defined?

Reply:
Where I describe the `type` [here](https://github.com/openid/sharedsignals/pull/101/files#diff-193e4a2db2f645f1b4f19c4a9f8facbef29f86e122a8da4c529542750a291a1cR567)